GliaWidgets/Sources/Layout/Layout+UIView.swift

```diff
@@ -0,0 +1,251 @@
+import UIKit
+
+extension UIView {
+    /// Defines available edges for establishing constratins.
```

Comment:
constraints, same thing in `Dimension`

Reply:
fixed - ee81f9a8cf5c703d98f5817e3caee70f2dea4044